bin/rake

```diff
@@ -5,15 +5,17 @@ load File.expand_path("../with_rubygems", __FILE__) if ENV["RGV"]
 
 require "rubygems"
 
-bundler_spec = Gem::Specification.load(File.expand_path("../../bundler.gemspec", __FILE__))
-bundler_spec.dependencies.each do |dep|
-  begin
-    gem dep.name, dep.requirement
-  rescue Gem::LoadError => e
-    $stderr.puts "#{e.message} (#{e.class})"
+unless ARGV[0] == "spec:deps"
```

Comment:
Could this be moved to Line 14 instead? It would make the diff be just a single line.

Reply:
Yes, but this version seem more correct to me since it doesn't do the unnecessary work of activating a bunch of dependencies that are most likely not installed? Anyways, I fine with your version if you have a strong opinion here. Just let me know and I'll update the PR (or go ahead and merge otherwise!).